.flake8

```diff
@@ -18,5 +18,8 @@ exclude =
 # https://www.youtube.com/watch?v=wf-BqAjZb8M&feature=youtu.be&t=260
 max-line-length=88
 
-# Allow longer lines for test fixtures
-per-file-ignores = tests/conftest.py:E501
+per-file-ignores =
+	# Allow longer lines for test fixtures
+	web/geosearch/tests/conftest.py:E501
+	# Allow unused imports because config is used as an object by Flask app
+	web/geosearch/datapunt_geosearch/*config.py:F401
```

Comment:
hiervoor zou je evt. ook een `# noqa` in de config.py kunnen zetten?

Reply:
Laten we niet te perfectionistisch doen op legacyprojecten.

Reply:
haha, ik had een noqa in conftest staan maar flake8 negeerde die als ik precommit --all-files runde